manuals/1.0/ja/apidoc.md

```
@@ -7,59 +7,73 @@ permalink: /manuals/1.0/ja/apidoc.html
+### 出力構造
 
-```php
-/**
- * {title}
- *
- * {description}
- *
- * {@link https://example.com/docs/auth 認証}
- */
-class Foo extends ResourceObject { }
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)